packages/cli/src/util/logger.js

```diff
@@ -4,34 +4,53 @@
 import chalk from 'chalk';
 
 const SEPARATOR = ', ';
+const TITLE = '[rn-cli]';
 
 const joinMessages = (messages: Array<string>) => messages.join(SEPARATOR);
 
+const getTimestamp = () => {
+  const date = new Date();
+  return `${date.getHours()}:${date.getMinutes()}:${date.getSeconds()}`;
+};
+
+const getHeader = () => chalk.gray(`${getTimestamp()} ${TITLE}`);
```

Comment:
Does this header bring value to the logger? I feel like it should be enabled only in verbose mode

Reply:
I was just adjusting the logger to similar I know. Got inspired by expo logger and others. We could enable it only in verbose mode and add millis to the pack. What about it?

Reply:
I'm not convinced to milliseconds (how often do you need that?) but the rest is cool

Reply:
Maybe we should drop the time. It is very useful when you have your own system, and logging different events throughout the time, and stream logs from a couple of microservices and then sort them. But in this case, it is useless. I deleted them and left only `[rn-cli]` header.

Reply:
@Esemesek use the time if in debug?

Reply:
@sidferreira Decided that time would be useless with current cli usage.